src/app/Navbar.tsx

```diff
@@ -0,0 +1,52 @@
+"use client"
+
+import Image from "next/image"
+import SolvroLogo from "./public/solvro_dark.png"
+import PromochatorLogo from "./public/promochator_logo.png"
+
+import {useState} from "react"
+
+export default function () {
```

Comment:
I know default export is convenient, but we should use named exports, so that it will be easier to maintain